### PR TITLE
Fix devtools-reps release documentation

### PR DESCRIPTION
**Pull request by nchevobbe**

### CODE REVIEW

File: packages/devtools-reps/RELEASE.md [hunk 1]
```diff
@@ -3,51 +3,45 @@
...
-  e. create a tag for v0.N+1.0 on github
-```
-
-After that any issue with the bundle should be addressed with a new build version. 
+1. **Prepare devtools-reps release**
+   1. Create a release branch on github
+   2. Update minor version in package.json, reset build version: devtools-reps-X.N.Y -> devtools-reps-X.N+1.0
+   3. Commit `release: bump version to devtools-reps-X.N+1.0`
+   4. Create a PR on Github for this release
+   5. Make sure you have a configs/local.json file, with `firefox.mcPath` pointing to your mozilla-central clone
```
Comment (juliandescottes) [inline packages/devtools-reps/RELEASE.md]: configs/local.json -> packages/devtools-reps/configs/local.json ?

File: packages/devtools-reps/RELEASE.md [hunk 2]
```diff
@@ -3,51 +3,45 @@
...
+   1. Update your mozilla-central clone to the latest
+   2. Create a new mercurial bookmark (or a local branch if you use git-cinnabar)
+   3. Navigate to the reps folder: `cd packages/devtools-reps`
+   4. And run `yarn copy-assets`
+   5. Create a new bug in [Developer Tools: Shared Components](https://bugzilla.mozilla.org/enter_bug.cgi?product=Firefox&component=Developer%20Tools%3A%20Shared%20Components)
+   6. Commit 'Bug XXX - devtools-reps v0.N+1.0: update reps bundle from GitHub;r=reviewer', where XXX is the number of the bug you created in `5.`.
+
+3. **Validate & cleanup**
+   1. push to try, test locally, submit for review etc ...
+   2. while try fails or some problem is detected, go back to devtools-reps, fix the issue, create a new bundle and go back to 2.a
```
Comment (juliandescottes) [inline packages/devtools-reps/RELEASE.md]: 2.a -> 2.1

File: packages/devtools-reps/RELEASE.md [hunk 3]
```diff
@@ -3,51 +3,45 @@
 Before you get started, you will need clones of devtools-reps and mozilla-central, as well as
```
Comment (juliandescottes) [inline packages/devtools-reps/RELEASE.md]: clones or devtools-reps -> clones of devtools-core

**Comment by nchevobbe** [comment]
Thanks for the review @juliandescottes :) Merging

**Pull request merged**